### ISSUE TITLE
Error not shown when deleting in-use image

### ISSUE THREAD

**Issue by lcarva** [crosbymichael/dockerui#186]
When attempting to remove an image from **Images** page by selecting an image and clicking on **Actions > Remove**, the operation fails because the image is in use by a container. This is expected behavior. However, there's no visual feedback to indicate an error occurred.

In browser's dev tools, Console tab displays a stack trace:

```
Error: [$resource:badcfg] Error in resource configuration for action `remove`. Expected response to contain an array but got an object
http://errors.angularjs.org/1.3.15/$resource/badcfg?p0=remove&p1=array&p2=object
    at angular.js:6
    at d.then.B.$resolved (angular.js:20)
    at i (angular.js:7)
    at angular.js:7
    at n.$eval (angular.js:8)
    at n.$digest (angular.js:8)
    at n.$apply (angular.js:8)
    at f (angular.js:7)
    at r (angular.js:7)
    at XMLHttpRequest.v.onload (angular.js:7)
```

And Network tab displays the following response for `DELETE` `/dockerapi/images/82a63ea3d16b3e8690ff2f37dbf46960ee939aba0b3e27c9b329657edf24f265`:

conflict: unable to delete 82a63ea3d16b (cannot be forced) - image is being used by running container 57f34405bcef

Would be great to at least display this error to user.

**Comment by kevana** [comment]
Related to #164, I with the API would return JSON errors...

**Comment by linquize** [comment]
This is really a problem

**Comment by lcarva** [comment]
I understand this is a docker api issue, but can we at least show an error when this occurs? Maybe add an interceptor via `$httpProvider` to handle this in a generic way globally?

**Comment by kevana** [comment]
I like the interceptor idea, but I won't have time until the weekend

**Comment by kevana** [comment]
Fixed by f353dc2c419664bc8aa5523bad6d9af7025106ee, there's probably some other error messages that don't start with "conflict.", let me know if you see any.
